### PR TITLE
INGM-366 Fix mypy issues due to adding py.typed in ingenialink

### PR DESCRIPTION
**Pull request by pzinemanas**
### Description

This PR fixes the issues that raised when adding the [py.typed file in ingenialink](https://github.com/ingeniamc/ingenialink-python/pull/336)

Fixes # INGM-336

### Type of change

- [x] Fix issues

### Tests
- [x] Run mypy and no error raised.

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingeniamotion tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

**Comment by pzinemanas** [comment]
The build will pass after merging https://github.com/ingeniamc/ingenialink-python/pull/336

**Review by WoodElv — APPROVED** [review]

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@develop
+ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@test-INGK-703
```
Comment (MartinALL91) [inline requirements/requirements.txt:1]: Revert this change

Comment (pzinemanas) [reply requirements/requirements.txt:1]: Yes, I'll remove it afterward finishing a test.

Comment (pzinemanas) [reply requirements/requirements.txt]: Done!

**Review by MartinALL91 — APPROVED** [review]
